core/management.c

```diff
@@ -96,7 +128,7 @@ coap_status_t handle_dm_request(lwm2m_context_t * contextP,
         {
             if (!LWM2M_URI_IS_SET_INSTANCE(uriP))
             {
-                result = object_create(contextP, uriP, message->payload, message->payload_len);
+                result = object_create(contextP, uriP, (char*)message->payload, message->payload_len);
```

Comment:
FYI: I plan to make a big cleanup and use only uint8_t * instead of this mix with char *